spec/controllers/games_controller_spec.rb

```diff
@@ -160,7 +160,7 @@
       game = Game.create!
```

Comment:
Hi @mscoutermarsh, I edited the tests to look like what you put in your comment in the other commit, but now I can't get them to pass. All 18 tests fail, and I'm returned a NoMethodError for .type. I tried removing the .type, to see what would happen, as I feel like it should still find the piece at that coordinate, but I get a failing message saying that it's nil. I edited the below test to try a different variation, and just put `Piece.find_by` rather than `game.pieces.find_by`, but no luck - still nil. I added a pry for more visibility when actually creating a new game, and both of the queries work. See attached screenshot in Slack.

Reply:
Hey @MattW216 - made you a video :smile:

http://cloud.mikeasaur.us/3S2Z2J3J2I0o

Watch that first and then for a solution: What I'm thinking is - We don't really need `user` on each piece. Since we have `color` already. So you could remove the `belongs_to :user` on pieces.

:party_parrot: